README.rst

```diff
@@ -24,12 +24,12 @@ Installation
 
 3. Install dependencies
 
-    .. code-block::
+   .. code-block::
       npm install
```

Comment:
Is this supposed to be visible in the normal view or just in an IDE?
![Screen Shot 2022-10-12 at 1 40 52 PM](https://user-images.githubusercontent.com/92897870/195411610-bb466075-0e76-44bd-8033-30d6ec0ace86.png)

Reply:
Oh thats very weird I think my editor formatted the code blocks out of existence